### DISCUSSION TITLE
NoIP - installing NoIP on BirdNET-Pi

### DISCUSSION THREAD

**Discussion by DD4WH** [mcguirepr89/BirdNET-Pi#149] [Q&A] [answered]
I tried to configure my BirdNET-Pi for access on the Internet.

I followed the instructions in the WIKI and followed this route, because I do not own a domain https://github.com/mcguirepr89/BirdNET-Pi/wiki/Sharing-Your-BirdNET-Pi#register-3-hostnames-and-1-group-with-noipcom

It works! And I can now connect via the Internet! 

However, in order to make it work in the longterm, the IP DUC has to be activated on the BirdNET-Pi. Therefore I issued `sudo ${HOME}/BirdNET-Pi/install_noip2.sh` in the web terminal. However, the command was not found. 

What am I doing wrong?

**Comment by DD4WH** [comment, marked as answer]
UPDATE: oh . . . 
that was easy:
The path is now: `sudo ${HOME}/BirdNET-Pi/scripts/install_noip2.sh`

IT WORKS NOW!

**Reply by mcguirepr89** [reply]
I'm excited to learn what your new URL is so I can take a look at your system!
